src/pk/ecc/ecc_verify_hash.c

```diff
@@ -76,11 +76,11 @@ int ecc_verify_hash_ex(const unsigned char *sig,  unsigned long siglen,
    }
    else if (sigformat == LTC_ECCSIG_RFC7518) {
       /* RFC7518 format - raw (r,s) */
-      i = mp_unsigned_bin_size(key->dp.order);
-      if (siglen != (2*i)) {
```

Comment:
I also preferred this version of the check TBH

Reply:
Yeah, I'm not convinced that {0x05, 0x01} should be a valid signature blob for ECDSA, let alone that it should be in a security/proofing library.

Reply:
I also don't like that it could be a valid signature, but it's a valid edge-case... :/ 

... and history has proven that there are always implementations who do worse than your implementation so this (`0x000..05...`) is a good example of Postel's Law IMO :)